docs/_data/formatters.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
if you merge latest master these generated files should be gone

Reply:
done